__tests__/endpoints.test.js

```diff
@@ -82,6 +82,51 @@ describe("Articles", () => {
             });
           });
       });
+      test("should correctly return articles based on query of sort_by and order", () => {
+        return request(app)
+          .get("/api/articles?sort_by=author&order=asc")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length > 1).toBe(true);
+            expect(body.articles[0]).toMatchObject({
+              author: "butter_bridge",
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            });
+          });
+      });
+      test("should correctly return articles based on query of topic", () => {
+        return request(app)
+          .get("/api/articles?sort_by=author&order=asc&topic=cats")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles[0]).toMatchObject({
+              author: expect.any(String),
+              title: expect.any(String),
```

Comment:
With this one, you can tell the topic is working, but cant really tell if the sort_by or order is working